NAS/running_s1floods_onnas.md

```diff
@@ -0,0 +1,79 @@
+# Running S1 Flood tutorials on NAS
+
+**Make sure you follow the steps in [initial setup](./initial_setup.md) and [TerraMind env creation](./terramind_env_creation.md) before running this tutorial!**
+
+This tutorial will walk you through running the [TerraMind Sentinel-1 flood segmentation tutorial](https://github.com/IBM/terramind/blob/main/notebooks/terramind_v1_base_sen1floods11.ipynb) as a job on the NAS system. The steps are as follows:
+
+1. Enter the environment
+1. Prepare input files
+1. Check the queue status
+1. Submit your job
+1. Monitor your job
+1. View stdout and stderr logs
+
+## Entering the environment
+SSH into into the Caebus front end (`ssh cfe`), navigate to your nobackup directory (`cd /nobackup/$USER`), make/enter a new terramind directory (`mkdir terramind && cd terramind`)
+
+```bash
+ssh cfe
+cd /nobackup/$USER
+mkdir terramind && cd terramind
+```
+## Prepare input files
+In your `terramind` directory `git clone` this repository, and copy the NAS submission script and the model python script to the current directory:
+
+```bash
+git clone https://github.com/ASFHyP3/terramind-docs.git
+cp terramind-docs/NAS/terramind_nas_submission_script.sh .
+cp terramind-docs/NAS/terramind_sen1floods.py .
+```
+
+In the same directory, download the input data:
+```bash
+curl -L "https://drive.usercontent.google.com/download?id={1lRw3X7oFNq_WyzBO6uyUJijyTuYm23VS}&confirm=xxx" -o sen1floods11_v1.1.tar.gz
+tar -xzf sen1floods11_v1.1.tar.gz
+rm sen1floods11_v1.1.tar.gz
+```
+
+You should now be ready to run the model.
+## Submit your job
+In NAS, we run jobs using the [Portable Batch System (PBS)](https://www.nas.nasa.gov/hecc/support/kb/portable-batch-system-(pbs)-overview_126.html). We submit jobs using the `qsub` command and monitor jobs using the `qstat` command.
+
+You will need to modify the submission script (`terramind_nas_submission_script.sh`) before running a job. Take a look at the submission script by calling `vi terramind_nas_submission_script.sh`. Lines starting with `##` are comments that describe what each set of lines is doing. Read through these. Next move to line 36 and change the email to your email. This will allow the NAS system to notify you of your job's status. 
```

Comment:
```suggestion
You will need to modify the submission script (`terramind_nas_submission_script.sh`) before running a job. Take a look at the submission script by calling `vi terramind_nas_submission_script.sh`. Lines starting with `# PBS` are configurations for PBS. Lines starting with `##` are comments that describe what each set of lines is doing.  Read through these comments. Next, move to line 36 and change the email to your email. This will allow the NAS system to notify you of your job's status. 
```